src/core/Akka/Util/MatchHandler/ILambdaExpressionCompiler.cs

```diff
@@ -22,15 +22,6 @@ internal interface ILambdaExpressionCompiler
         /// <param name="expression">The expression to compile</param>
         /// <returns>A delegate containing the compiled version of the lambda.</returns>
         Delegate Compile(LambdaExpression expression);
-
-#if !CORECLR
```

Comment:
Byterot, these codes were never compiled before the cleanup (since we moved to single `netstandard2.0` platform target)